src/briefcase/integrations/file.py

```diff
@@ -22,7 +22,7 @@
 from briefcase.integrations.base import Tool, ToolCache
 
 if TYPE_CHECKING:
-    from typing import Iterable, Sequence
+    from collections.abc import Iterable, Sequence
```

Comment:
I'm a bit curious...why gate this with `TYPE_CHECKING`?

Reply:
Most of these changes are what pyupgrade generated automatically; if it was in `TYPE_CHECKING` before, it is after as well. There's a *really* minor import speed gain... but no functional reason that I can think of.